sahi/predict.py

```diff
@@ -340,18 +343,18 @@ def agg_prediction(result: PredictionResult, thresh):
 
 
 def predict(
-    detection_model: DetectionModel = None,
+    detection_model: Optional[DetectionModel] = None,
     model_type: str = "mmdet",
```

Comment:
we should update it to ultralytics since its more stable and has much larger userbase. wdyt? @Dronakurl